static/app/views/performance/transactionSummary/transactionOverview/eapChartsWidget.tsx

```diff
@@ -95,28 +96,26 @@ export function EAPChartsWidget({transactionName}: EAPChartsWidgetProps) {
 
   return (
     <Widget
-      Title={<Widget.WidgetTitle title={title} />}
-      Actions={
-        <Widget.WidgetToolbar>
-          <Widget.WidgetDescription title={title} description={description} />
-        </Widget.WidgetToolbar>
-      }
-      Visualization={visualization}
-      Footer={
-        <FooterContainer>
+      Title={
+        <WidgetTitleWrapper>
           <CompactSelect
             options={options}
             value={selectedWidget}
             onChange={option => setSelectedWidget(option.value as EAPWidgetType)}
+            triggerProps={{borderless: true, size: 'md'}}
           />
-        </FooterContainer>
+          <Widget.WidgetDescription title={title} description={description} />
```

Comment:
👀 The description is usually better suited to be in the `Actions=` slot! That way it slides over to the right naturally and you don't need to add your own flex container around things. Even better if you wrap it in `Widget.WidgetToolbar`, so it will create the correct spacing if you ever add more items

Reply:
Ah thanks! I had it initially in the toolbar but was messing around with designs and forgot to move it back. Will do that now 👍